cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -759,10 +772,10 @@ def test_access_child_services_from_the_my_service():
     pass
 
 
-@pytest.mark.meta(coverage=[1684575])
-@pytest.mark.manual
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1684575])
 @pytest.mark.tier(2)
-def test_load_values_on_init_option_service_dialog_element():
```

Comment:
Remove tier marker. It is already defined globally: https://github.com/ManageIQ/integration_tests/pull/9995/files#diff-faeac2d3d489ac01e0ebae6983c9c3cbR21

Reply:
@dgaikwad any update on this comment?